apps/crn-frontend/src/shared-research/export.ts

```diff
@@ -5,22 +5,58 @@ import {
   GetListOptions,
   htmlToCsvText,
 } from '@asap-hub/frontend-utils';
-import { ListResponse, ResearchOutputResponse } from '@asap-hub/model';
+import {
+  ListResponse,
+  ResearchOutputResponse,
+  ResearchOutputVersion,
+} from '@asap-hub/model';
 import { isInternalUser } from '@asap-hub/validation';
 import { Stringifier } from 'csv-stringify/browser/esm';
 
 export const MAX_ALGOLIA_RESULTS = 10000;
 // Max Complexity 11000. Research Outputs are the most complex entity. 554 11000/554 = 19.8. 15 seems safe.
 export const MAX_CONTENTFUL_RESULTS = 15;
 
+type FirstVersionCSV = {
+  firstVersionTitle: string;
+  firstVersionType?: string;
+  firstVersionRrid?: string;
+  firstVersionAccession?: string;
+  firstVersionLink?: string;
+};
+
 type ResearchOutputCSV = Record<
   keyof Omit<
     ResearchOutputResponse,
-    'team' | 'descriptionMD' | 'usageNotesMD' | 'versions' | 'shortDescription'
-  >,
+    'team' | 'descriptionMD' | 'usageNotesMD' | 'versions'
+  > &
+    keyof FirstVersionCSV,
   CSVValue
 >;
 
+const getFirstVersionData = (
+  versions: Array<ResearchOutputVersion>,
+): FirstVersionCSV => {
+  if (versions[0]) {
+    const { title, type, rrid, accession, link } = versions[0];
+    return {
+      firstVersionTitle: title,
+      firstVersionType: type,
+      firstVersionRrid: rrid,
+      firstVersionAccession: accession,
```

Comment:
I believe we need to modify the query to return te rrid and accession in the version
https://github.com/yldio/asap-hub/blob/master/packages/contentful/src/crn/queries/research-outputs.queries.ts#L216:L225

Reply:
Nice catch! I didn't realise it cause the other fields were retrieved and I knew some were only recently added.

Reply:
If I’m not mistaken, we also need to add those fields to the data provider to ensure they appear in the export
https://github.com/yldio/asap-hub/blob/master/apps/crn-server/src/data-providers/contentful/research-output.data-provider.ts#L274 :)